qutebrowser/browser/webkit/network/qutescheme.py

```diff
@@ -261,3 +261,19 @@ def qute_pdfjs(_win_id, request):
             "pdfjs resource requested but not found: {}".format(e.path))
         raise QuteSchemeError("Can't find pdfjs resource '{}'".format(e.path),
                               QNetworkReply.ContentNotFoundError)
+
+
+@add_handler('bookmarks')
+def qute_bookmarks(_win_id, request):
+    """Handler for qute:bookmarks. Show a list of all quickmarks / bookmarks"""
+
```

Comment:
Hmm, I might actually turn that linting check off at some point, but no blank line here for now please to keep the linters happy.